infra-as-code/bicep/modules/resourceGroup/resourceGroup.bicep

```diff
@@ -1,15 +1,17 @@
 targetScope = 'subscription'
 
-@description('Azure Region where Resource Group will be created.  No Default')
+@description('Azure Region where Resource Group will be created.')
 param parLocation string
 
-@description('Name of Resource Group to be created.  No Default')
+@description('Name of Resource Group to be created.')
+@minLength(2)
```

Comment:
Breaking, if exceeded maxLength

Reply:
Although technically. Nobody could deploy if they broke the length as it's a platform limit